homeassistant/components/control4/light.py

```diff
@@ -185,13 +187,15 @@ def _create_api_object(self):
     @property
     def is_on(self):
         """Return whether this light is on or off."""
-        return self.coordinator.data[self._idx]["value"] > 0
+        if self._is_dimmer:
+            return self.coordinator.data[self._idx][CONTROL4_DIMMER_VAR] > 0
+        return self.coordinator.data[self._idx][CONTROL4_NON_DIMMER_VAR] > 0
```

Comment:
not a bugfix the new function implements a different data structure where we need to access the "value" by name since we will be supporting more than one variable value to be stored.